lms/templates/student_account/login_and_register.html

```diff
@@ -22,6 +22,7 @@
             newrelic.addPageAction('xfinished');
         }
     </%static:require_module>
+    <script type="text/javascript" src="${static.url('js/student_account/tos_modal.js')}"></script>
```

Comment:
`SiteConfiguration` is a database model. You would set the value using [Django admin](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/sites/configure_site.html).